management/examples/createVM/createVM_test.go

```diff
@@ -0,0 +1,9 @@
+package createVM
```

Comment:
please delete *_test.files under example/ and turn them into `main()` functions, in separate packages. this file is very unidiomatic (though I know you didn't add it)